analysis.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import numpy as np
```

Comment:
I see that you already have numpy imported. Maybe try using numpy to make a log chart of the data for readability? :)